llvm/test/CodeGen/AArch64/nonwindows-coff-mc-error.ll

```diff
@@ -0,0 +1,5 @@
+; RUN: not llc -mtriple=aarch64-unknown-linux-coff -filetype=null %s 2>&1 | FileCheck %s
```

Comment:
This can be merged into the other unsupported object file format test you are adding. Perhaps rename the test if necessary.